src/bidiMapper/domains/context/context.ts

```diff
@@ -28,7 +28,6 @@ export class Context {
   _targetInfo?: Protocol.Target.TargetInfo;
   _sessionId?: string;
 
-  private _dummyContextObjectId: string = '';
```

Comment:
The was a problem: after navigation the world is recreated, and the `dummyContextObject` was destroyed. So it should be either created after each navigation or on each evaluation call. Anyway, it should be removed as part of #52 and #65